openlibrary/core/models.py

```diff
@@ -283,8 +283,7 @@ def is_in_private_collection(self):
 
     def in_borrowable_collection(self):
         collections = self.get_ia_collections()
-        return ('lendinglibrary' in collections
-                and not self.is_in_private_collection())
+        return 'inlibrary' in collections and not self.is_in_private_collection()
```

Comment:
It does look like `lendinglibrary` is deprecated ( https://archive.org/search.php?query=collection%3Alendinglibrary ); but let's deal with it in a future PR so we don't have to update tests (Issue already exists: #2901).

```suggestion
        return ('lendinglibrary' in collections or 'inlibrary' in collections
                    and not self.is_in_private_collection())
```